interface-definitions/service-zabbix-agent.xml.in

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="service">
+    <children>
+      <node name="zabbix-agent" owner="${vyos_conf_scripts_dir}/service_zabbix_agent.py">
+        <properties>
+          <help>Zabbix-agent settings</help>
+          <priority>380</priority>
+        </properties>
+        <children>
+          <leafNode name="directory">
+            <properties>
+              <help>Folder containing individual Zabbix-agent configuration files</help>
+              <constraint>
+                <validator name="file-path" argument="--directory"/>
+              </constraint>
+            </properties>
+          </leafNode>
+          <node name="limits">
+            <properties>
+              <help>Limit settings</help>
+            </properties>
+            <children>
+              <leafNode name="buffer-send">
+                <properties>
+                  <help>Do not keep data longer than N seconds in buffer</help>
+                  <valueHelp>
+                    <format>u32:1-3600</format>
+                    <description>Seconds</description>
+                  </valueHelp>
+                  <constraint>
+                    <validator name="numeric" argument="--range 1-3600"/>
+                  </constraint>
+                  <constraintErrorMessage>Buffer-send must be between 1 and 3600 seconds</constraintErrorMessage>
```

Comment:
I believe we should keep option names lowercase, but see above — I think the name of this option can also be better.

Reply:
Done! Replaced to `buffer-flush-interval`